client/src/metrics.rs

```diff
@@ -126,12 +136,40 @@ pub(crate) fn svc_operation(op: String) -> MetricKeyValue {
     MetricKeyValue::new(KEY_SVC_METHOD, op)
 }
 
+pub(crate) fn status_code_kv(code: Code) -> MetricKeyValue {
+    MetricKeyValue::new(KEY_STATUS_CODE, code_as_screaming_snake(&code))
+}
+
+/// This is done to match the way Java sdk labels these codes (and also matches gRPC spec)
+fn code_as_screaming_snake(code: &Code) -> &'static str {
```

Comment:
Pretty annoying https://docs.rs/tonic/latest/tonic/enum.Code.html doesn't give you the raw enum from https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto so that you can use the proper enum name